### ISSUE TITLE
Angular 5 support

### ISSUE THREAD

**Issue by MedFermi** [Zak-C/ngx-loading#23]
The library doesn't seem to be compatible with Angular 5.
Would that be possible to change that?
Thanks!

**Comment by beneschmidt** [comment]
For me it's working with Angular 5. What's the problem with it? Is it throwing any errors?

**Comment by MedFermi** [comment]
It's actually working for me, as well.
I just get those warning when running npm install:
npm WARN ngx-loading@1.0.8 requires a peer of @angular/core@^4.0.0 but none was installed.
npm WARN ngx-loading@1.0.8 requires a peer of @angular/common@^4.0.0 but none was installed.

**Comment by dfederm** [comment]
I'm getting these warnings when running `npm install`:
```
npm WARN ngx-loading@1.0.9 requires a peer of @angular/core@^4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-loading@1.0.9 requires a peer of @angular/common@^4.0.0 but none is installed. You must install peer dependencies yourself.
```

Functionally it's working though, at least for my app, so optimistically only the `package.json` needs to be updated to declare its support for Angular 5 and it'll just work.

**Comment by Zak-C** [comment]
Hi, 

As discussed, the library is compatible with Angular 5. 

I have just released v1.0.14 of ngx-loading, which will remove the warnings you were experiencing when installing the library via "npm install". 

Thanks
